modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -1072,18 +1072,8 @@ def obj.class
           }
         end
         let(:mvi_profile) { build(:mvi_profile) }
-        let(:mvi_profile_response) do
-          MPI::Responses::FindProfileResponse.new(
```

Comment:
Is there a reason to use create, rather than build?

Reply:
Since I was changing that other spec I decided just to change this to `build` for these. Doesn't appear to have an effect and it's prob better practice as a general rule of thumb to prefer `build` unless `create` is necessary